sass/tji/_donations.scss

```diff
@@ -0,0 +1,150 @@
+/**
+ * Styles for the TJI Donations page
+ */
+
+.donation-form {
+	
+}
+
+.donation-form__label {
```

Comment:
🙏 for sticking to the BEM naming convention!!!!

Reply:
👍